global.json

```
@@ -1,4 +1,11 @@
 {
+  "sdk": {
+    "paths": [
+      ".dotnet",
+      "$host$"
+    ],
+    "errorMessage": "The .NET SDK is not installed or is not configured correctly. Please run ./build to install the correct SDK version locally."
```

Consider using the same message as in Arcade so that we are consistent in our repos: https://github.com/dotnet/arcade/blob/e554a518e2d93daee29a8aad27fd471a8804d8e1/global.json#L9

```suggestion
    "errorMessage": "The required .NET SDK wasn't found. Please run ./eng/common/dotnet.cmd/sh to install it."
```

This also uses eng/common/dotnet.cmd/sh which just takes care of downloading the .NET SDK.

---

Why point to the nested eng script instead of the 'normal' entrypoint for this repo, which chains to that script?